client/src/App.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+import { Input, Row } from 'react-materialize';
+
+// import D3 from './d3';
+import Chart from './Component/Chart';
+import TopNav from './Component/TopNav';
+import BottomNav from './Component/BottomNav';
+import './App.css';
+
+// cannot use route '/data', seems to be a reserved word?
+const callApi = async (route) => {
+  const response = await fetch(route);
+  const body = await response.json();
+  return body;
+};
+
+class App extends Component {
+  // can I rewrite the state to { stateName: '', statePrice: 0 }?
```

Comment:
Not sure what this question means.  Do you mean here in the constructor, or elsewhere?

If in constructor, yes, you can just add `statePrice: 0,` to the block starting on line 21.

If later, you'd need to use `this.setState` to update the state.

Reply:
Thank you! That's my old comment :) I may need to rethink of the `state` of `App` component.